lib/ErrorManager.php

```diff
@@ -20,9 +20,48 @@ public function __construct(Configuration $configuration)
         $this->configuration = $configuration;
     }
 
+    public function addError(Error $error): void
+    {
+        $this->errors[] = $error;
+
+        if (! $this->configuration->isSilentOnError()) {
+            echo '/!\\ ' . $error->__toString() . "\n";
+        }
```

Comment:
If we want to make an abstraction of the error handling in this library (which I agree with), I would say that we also move this code into an `StdErrorReporter` or the like (so that we can also e.g. add an `GitHubErrorReporter`).

Reply:
I pushed some changes without the abstraction. When we are adding a third format, then I think it starts to make sense with an abstraction. Right now it is just a simple if statement.